### ISSUE TITLE
Indicate when song(s) can't accept custom tags

### ISSUE THREAD

**Issue by ptitjes** [quodlibet/quodlibet#2601]
Before 3.9 (I believe), we were able to add custom tags to a song in the `Properties` window of a song. The text box to enter tag name even had completion based on the existing tags in library.

Now the `Tag` name selection is a drop-down menu. I feel this is a regression for at least two reasons :
- You can't enter a custom tag name
- Now, even if you want title, just pressing `T` while the drow-down is focused does absolutely nothing

I'm really surprised as QL was the best librarian ! I can't handle my library without custom tag.

**Comment by lazka** [comment]
Afaik the dropdown is shown in case one of the selected formats only supports a fixed set of tags.

It would probably be best if it was shown "red" + an info text "one of the selected format does not support the tag 'foo'" or something.

**Comment by declension** [comment]
>> Afaik the dropdown is shown in case one of the selected formats only supports a fixed set of tags.

Yes, that's definitely the case here - `m4a` will exhibit this behaviour.

>> It would probably be best if it was shown "red" + an info text "one of the selected format does not support the tag 'foo'" or something.

Agreed - a warning would be nice. Not sure how we'd predict the tag they wanted as it's no longer free text entry but the constraint has come up before (as confusing) IIRC, so it'd be good to avoid confusing users.


> Now, even if you want title, just pressing T while the drow-down is focused does absolutely nothing
Yes, it would be nice if the Combox reacted to "search" / keypress. Perhaps, given the other part is expected behaviour, we should re-purpose this issue for that enhancement...

**Comment by ptitjes** [comment]
Sorry that I shouted too early... Indeed, the file is a m4a.

I think the proposal of @lazka would indeed make that more clear.

I will rename this issue accordingly.
